src/modules/trading-view/dtos/bars.response.ts

```diff
@@ -0,0 +1,81 @@
+import { Type } from 'class-transformer';
+import { IsEnum, IsInt, IsNotEmpty, IsOptional } from 'class-validator';
+import { PriceCandlesResolutions } from 'src/modules/analytics/models/query.args';
+
+export const supportedResolutions = [
+    '1',
+    '5',
+    '15',
+    '30',
+    '60',
+    '240',
+    '1D',
+    '7D',
+    '1M',
+];
+
+export enum TradingViewResolution {
+    MINUTE_1 = '1',
+    MINUTE_5 = '5',
+    MINUTE_15 = '15',
+    MINUTE_30 = '30',
+    HOUR_1 = '60',
+    HOUR_4 = '240',
+    DAY_1 = '1D',
+    DAY_7 = '7D',
+    MONTH_1 = '1M',
+}
+
+export class BarsResponse {
+    s: string;
```

Comment:
maybe add comments for each field? the short names aren't descriptive

Reply:
Added in https://github.com/multiversx/mx-exchange-service/pull/1386